benchmarks/instruction_counts/core/utils.py

```diff
@@ -13,9 +14,8 @@
 def get_temp_dir() -> str:
     global _TEMPDIR
     if _TEMPDIR is None:
-        temp_dir = tempfile.mkdtemp()
-        atexit.register(shutil.rmtree, path=temp_dir)
-        _TEMPDIR = temp_dir
+        _TEMPDIR = _make_temp_dir(prefix="instruction_count_microbenchmarks", gc_dev_shm=True)
```

Comment:
Just curious are the temp directories cleaned up? The cleanup code seems not part of this PR?

Reply:
It's up to the user to clean up. (Mirroring the contract of `tempfile.mkdtemp`)